langchain-core/src/language_models/chat_models.ts

```diff
@@ -185,13 +185,14 @@ export abstract class BaseChatModel<
       const runManagers = await callbackManager_?.handleChatModelStart(
         this.toJSON(),
         [messages],
-        undefined,
+        runnableConfig.runId,
         undefined,
         extra,
         undefined,
         undefined,
         runnableConfig.runName
       );
+      delete runnableConfig.runId;
```

Comment:
Instead of mutating the object, would it make more sense to do like:

```ts
const { runId, ...childConfig } = runnableConfig;
```

?

Reply:
Realized I could remove the deletion here, `runnableConfig` isn't used after this.